clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBServerConfig.java

```diff
@@ -219,7 +219,6 @@ public class RocksDBServerConfig {
 
   private final long rocksDBBlockCacheSizeInBytes;
   private final long rocksDBRMDBlockCacheSizeInBytes;
-  private final long rocksDBBlockCacheCompressedSizeInBytes;
```

Comment:
Is this removed in the latest version of RocksDB?

Reply:
yes.